qpretrieve/fourier/base.py

```diff
@@ -70,9 +75,13 @@ def __init__(self, data, subtract_mean=True, padding=2, copy=True):
         else:
```

Comment:
Here we check and allow complex data. But in the docstring above (originally), we specify real-valued inputs. In what situation would we take complex data?